rules/RESPONSE-951-DATA-LEAKAGES-SQL.conf

```diff
@@ -286,7 +286,7 @@ SecRule RESPONSE_BODY "@rx (?i)(?:System\.Data\.OleDb\.OleDbException|\[Microsof
 # (consult https://coreruleset.org/docs/development/regex_assembly/ for details):
 #   crs-toolchain regex update 951230
 #
-SecRule RESPONSE_BODY "@rx (?i)(?:supplied argument is not a valid |SQL syntax.*)MySQL|Column count doesn't match(?: value count at row)?|mysql_fetch_array\(\)|on MySQL result index|You have an error in your SQL syntax(?:;| near)|MyS(?:QL server version for the right syntax to use|qlClient\.)|\[MySQL\]\[ODBC|(?:Table '[^']+' doesn't exis|valid MySQL resul)t|Warning.{1,10}mysql_(?:[\(\)_a-z]{1,26})?|(?:ERROR [0-9]{4} \([0-9a-z]{5}\)|XPATH syntax error):" \
+SecRule RESPONSE_BODY "@rx (?i)s(?:upplied argument is not a valid |ql syntax.*)mysql|column count doesn't match(?: value count at row)?|mysql(?:_fetch_array\(\)| server version for the right syntax to use|client\.)|on mysql result index|you have an error in your sql syntax(?:;| near)|\[mysql\]\[odbc|(?:table '[^']+' doesn't exis|valid mysql resul)t|warning.{1,10}mysql_(?:[\(\)_a-z]{1,26})?|(?:error [0-9]{4} \([0-9a-z]{5}\)|xpath syntax error):" \
```

Comment:
I wonder if ignore case is wise in this situation 🤔

⚠️ Maybe we want to actually remove the ignore case here.

Reply:
I don't think that it matters. The strings are very unique already, making the match case insensitive doesn't increase the probability for FPs, IMO. We also lose a bit of information in the source files, of course. I'll remove the flag for both files.